index.html

```diff
@@ -179,22 +182,21 @@
       <p>
         With an active, public development roadmap, seL4 continues to solidify its
         position as the leading secure operating system and the industry
-        standard for verified software.
+        standard for verified software. Evolution is a constant in the seL4 ecosystem.
```

Comment:
I'm not sure I'm convinced here. evolution+constant sounds a bit like an oxymore, even if I know what you mean. And I didn't see at first why you wanted that addition and what you meant by "in the seL4 ecosystem" but I think it's meant to introduce the several levels below... 
Maybe "Evolution happens at every level of the seL4 ecosystem."?

Reply:
It's not an oxymoron, change is indeed the only constant. Unless nothing changes, but that would also be a change :-)

Happy to change it (😁), but I don't like `happens`. Maybe `Evolution drives every level of the seL4 ecosystem`?

Reply:
Oooh, "drives" is nice!

Reply:
Looks like we commented at the same time, I was replying to June :-)